examples/stripe-integration/app/routes/buy.tsx

```diff
@@ -0,0 +1,53 @@
+import { loadStripe } from "@stripe/stripe-js";
+import { Form, useActionData, useLoaderData } from "remix";
+import type { ActionFunction } from "remix";
+import { getStripeSession } from "~/utils/stripe.server";
+
+type returnData = {
```

Comment:
This is very unusual to have your loader and your action return the same data type. Could you have a `LoaderData` type and an `ActionData` type? In this example they don't technically even return the same data, so it doesn't make sense to use the same data type.

Reply:
Thanks much Kent for reviewing, fixed